libraryAPI/pkg/repository/book_csv.go

```diff
@@ -0,0 +1,69 @@
+package repository
+
+import (
+	"errors"
+	"library/pkg/model"
+	"os"
+
+	"github.com/gocarina/gocsv"
+)
+
+type bookCSVRepository struct {
+	file string
+}
+
+func NewCSVBookRepository(file string) BookRepository {
+	return &bookCSVRepository{
+		file: file,
+	}
+}
+
+func (r *bookCSVRepository) CreateBooks(books []model.Book) (int, error) {
+	booksFile, err := os.OpenFile(r.file, os.O_RDWR|os.O_CREATE, os.ModePerm)
+	if err != nil {
+		return 0, err
+	}
+
+	defer booksFile.Close()
```

Comment:
let's try to move it to the main and provide not `filepath` to the repo, but an actual connection to the file, to avoid opening a connection on each operation

Reply:
if there will be any problems with file sync - please tell me and i'll try to help you with it